src/main/kotlin/com/pubnub/api/managers/PresenceEventEngineManager.kt

```diff
@@ -0,0 +1,30 @@
+package com.pubnub.api.managers
+
+import com.pubnub.api.eventengine.EffectDispatcher
+import com.pubnub.api.eventengine.Event
+import com.pubnub.api.eventengine.EventEngineManager
+import com.pubnub.api.eventengine.Sink
+import com.pubnub.api.presence.eventengine.PresenceEventEngine
+import com.pubnub.api.presence.eventengine.effect.PresenceEffectInvocation
+import com.pubnub.api.presence.eventengine.event.PresenceEvent
+
+class PresenceEventEngineManager(
```

Comment:
This class is the same as the `SubscribeEventEngineManager`. I think we should have just one implementation